set.mm

```diff
@@ -629404,20 +629404,25 @@ standardize vectors to a length (norm) of one, but such definitions require
   To achieve the expected results, the following substitutions should be
   additionally performed whenever possible:
 
-  - ~ sb8eu -> ~ sb8euv
-  - ~ sbie -> ~ sbiev
+  - ~ spim -> ~ spimfv
+  - ~ spimev -> ~ spimevw
+  - ~ spv -> ~ spvv
+  - ~ spei -> ~ speivw
   - ~ chvar -> ~ chvarfv
-  - ~ nfsb -> ~ nfsbv
-  - ~ nfmo -> ~ nfmov
-  - ~ sb8 -> ~ sb8v
   - ~ chvarv -> ~ chvarvv
-  - ~ rgen2a -> ~ rgen2
-  - ~ spv -> ~ spvv
+  - ~ cbvexv -> ~ cbvexvw
+  - ~ sbie -> ~ sbiev
+  - ~ sbid2v -> ~ sbid2vw
+  - ~ sb8 -> ~ sb8v
+  - ~ sb8e -> ~ sb8ev
+  - ~ nfsb -> ~ nfsbv
   - ~ cbval2 -> ~ cbval2v
   - ~ cbvex2 -> ~ cbvex2v
-  - ~ dral1 -> ~ dral1v
-  - ~ drex1 -> ~ drex1v
-  - ~ drnf1 -> ~ drnf1v
+  - ~ nfmod -> ~ nfmodv
+  - ~ nfmo -> ~ nfmov
+  - ~ sb8eu -> ~ sb8euv
+  - ~ moexexv -> ~ moexexvw
+  - ~ rgen2a -> ~ rgen2
 
   Note: ~ cbval2w is identical to ~ cbval2v , but has shorter proof.
```

Comment:
Checked. Can you move cbval2w to be the new cbval2v ("Proof shortened by GG", etc.) and keep the current one as cbval2vOLD ?

Reply:
Done in #3764